app/models/planning_application.rb

```diff
@@ -1,8 +1,18 @@
+# frozen_string_literal: true
+
 class PlanningApplication < ApplicationRecord
   belongs_to :local_authority
   belongs_to :property
 
   delegate :address, to: :property
 
-  validates :reference, :area, :proposal_details, :received_at, :decision, :decision_issued_at, :local_authority, :property, presence: true
+  validates(
+    :reference,
+    :area,
+    :proposal_details,
+    :received_at,
+    :decision,
+    :decision_issued_at,
+    presence: true
+  )
```

Comment:
I liked that but I am not a big fun in `()`, what do you think?

Reply:
() away! 🚀